pkg/client/client_test.go

```diff
@@ -20,6 +21,7 @@ func TestSendReports(t *testing.T) {
 				fernUrl:     mockFernReporter.URL,
 				projectName: testProjectName,
 				filePattern: reportsCombinedPattern,
+				tags:        "test,tagtest,9=-+_",
```

Comment:
Could we make this a variable/constant, like `testProjectName`?

Reply:
Moved to `exampleTags`